x-pack/plugins/fleet/server/collectors/agent_collectors.ts

```diff
@@ -117,6 +122,9 @@ export const getAgentData = async (
           policies: {
             terms: { field: 'policy_id' },
           },
+          os: {
+            terms: { field: 'local_metadata.os.full.keyword' },
```

Comment:
In the issue description it mentioned os platform and os version, so adding the telemetry on full looks good, just wanted to confirm that this is what we want (not version and platform separately).

Reply:
Good catch, just updated everything